### PR TITLE
BinaryBuilder in Mac and Linux

### PR DESCRIPTION
**Pull request by juan-pablo-vielma**

### CODE REVIEW

**Comment by juan-pablo-vielma** [comment]
Looks like CXX requires Julia v1.0

File: deps/build.jl [hunk 1]
```diff
@@ -1,15 +1,61 @@
-using Libdl
-
-using BinDeps
-# If I do not do using CxxWrap, when BinDeps try dlopen on libsdpawrap.so, I get
-# ERROR: could not load library "~/.julia/v0.5/SDPA/deps/usr/lib/libsdpawrap"
-# libcxx_wrap.so.0: cannot open shared object file: No such file or directory
+using BinaryProvider # requires BinaryProvider 0.3.0 or later
```
Comment (blegat) [inline deps/build.jl]: In REQUIRE, it says that it needs BP v0.5.3 or later

File: src/SDPA.jl [hunk 2]
```diff
@@ -2,6 +2,10 @@ module SDPA
 
 using CxxWrap
 
+# This 'using' is required to suppress a warning about Clp not having Libdl in its
```
Comment (blegat) [inline src/SDPA.jl]: Clp -> SDPA

Comment (juan-pablo-vielma) [reply src/SDPA.jl]: updated

File: deps/build.jl (hunk 1 above)
Comment (juan-pablo-vielma) [reply deps/build.jl]: updated

**Review by blegat — APPROVED** [review]
Fantastic !

**Pull request merged**